### DISCUSSION TITLE
GIA just not downloading programs

### DISCUSSION THREAD

**Discussion by jim-ip** [Ascoware/get-iplayer-automator#421] [General]
Hi there. Hopefully this isn't an FAQ. If there's a better/more appropriate place to ask then please point me in the right direction! 

I'm running the current build of GIA on a 2015 MacBook Pro (OSX Catalina 10.15.7). I've not used GIA for a while, and have a feeling that I stopped using it because of this. However, I now want to use it and can't get it to download programs. 

Loading program indexes is fine. When I try to download programs, the log says:

INFO: Loading proxy settings...
INFO: No proxy to load
INFO: Proxy load complete.
INFO: No proxy will be used
AppController: Starting Downloads

and then it just doesn't download anything.

Any and all suggestions much appreciated.

**Comment by skovatch** [comment]
Please turn on 'Verbose' mode in the preferences, start downloads again, and post the output from that.

**Comment by jim-ip** [comment]
Thanks for the quick reply. Here you go...

Get iPlayer Automator 1.24.3 Initialized.
<AppController: 0x7fa966c1bd60>: Updating Programme Index Feeds...

/Applications/Get iPlayer Automator.app/Contents/Resources/get_iplayer/perl/bin/get_iplayer
-e14400
--type=tv,radio
--refresh
--nopurge
--profile-dir=/Users/jim/Library/Application Support/Get iPlayer Automator
.*
INFO: Indexing tv programmes (concurrent)
INFO: Added 0 tv programmes to cache
INFO: Indexing radio programmes (concurrent)
INFO: Added 0 radio programmes to cache
INFO: 22457 matching programmes
<AppController: 0x7fa966c1bd60>: Index Updated.


INFO: Loading proxy settings...
INFO: No proxy to load
INFO: Proxy load complete.
INFO: No proxy will be used
AppController: Starting Downloads

**Comment by skovatch** [comment]
What program are you trying to download? It looks like there's nothing in your queue, but you shouldn't be able to start downloads when that happens.

**Reply by jim-ip** [reply]
It was a set of radio shows. I just deleted them, restarted GIA, and added a new program via "Use current webpage". What that gave me was the newly added program with a blank line above it in the download queue. I clicked on the start button and (as before) nothing happened. However, I then clicked on the "Stop" button, selected and deleted the blank first line in the download queue, and clicked on the download button. At that point, it started to download the new program. 

Interestingly, I hadn't previously noticed a blank entry in the queue (or at least not a blank entry at the top of the queue). I'm wondering if there was a blank entry in the queue, but that just wasn't noticeable because it was at the bottom and the new items from Series Link were being added above it.

**Reply by skovatch** [reply]
Thanks, this is extremely helpful! I’ve seen reports like this with no clue as to how to proceed but now I think I know where to look.

**Comment by skovatch** [comment]
Which page did you use when this happened? That will save me some time.

**Reply by jim-ip** [reply]
Does this log give you what you need? [https://pastebin.com/KC6wSWae](url)

**Reply by skovatch** [reply]
Link gives a 404, but the BBC iplayer link is good enough

**Reply by jim-ip** [reply]
That's strange. The link points at GitHub - the text of the pastebin link is correct.

The iplayer link is: https://www.bbc.co.uk/iplayer/episode/b00gd1rn/wallace-and-gromit-in-the-curse-of-the-wererabbit
